### ISSUE TITLE
nyxus.Featurize() appears to switch intensity_image and mask_image names

### ISSUE THREAD

**Issue by TimMonko** [PolusAI/nyxus#210]
First off, great library! I am glad I found this, as many of the measurements are useful and unavailable elsewhere (at least easily).

Second, when I pass `intensity_names` and `label_names` to `Nyxus.featurize()`, the pandas column names seem intuitively switched. 

```
stats = nyx.featurize(
    intensity_images=fo, 
    label_images=label_image,
    intensity_names=['FerroOrange'],
    label_names=['Labels']
)
```
The 'intensity_image' column is populated with 'labels' and the 'mask_image' column with 'FerroOrange'.
I would expect the opposite, even without the specific intensity_name var in featurize, I think of masks as the label as well. 

Perhaps this is intentional, and just confusion of my intuition! But even passing nothing to those arguments also leads to the confusing `'intensity_image'='Segmentation0'` and `'mask_image'='Intensity0'`. 

`nyxus.__version__='0.7.5'`

**Comment by sameeul** [comment]
Thanks for reporting this. I looked at the code and found the issue. We will push a fix soon.

**Comment by sameeul** [comment]
Fixed in v0.8.1
